content/source/docs/enterprise/install/installer.html.md

```diff
@@ -192,6 +192,7 @@ If your instance can access the Internet, use the Online install mode.
 1. From a shell on your instance:
     * To execute the installer directly, run `curl https://install.terraform.io/ptfe/stable | sudo bash`
     * To inspect the script locally before running, run `curl https://install.terraform.io/ptfe/stable > install.sh` and, once you are satisfied with the script's content, execute it with `sudo bash install.sh`.
+    * To install on RedHat Enterprise Linux where pre-installation of Docker is necessary, run `curl https://install.terraform.io/ptfe/stable > install.sh` and, once you are satisfied with the script's content, execute it with `sudo bash install.sh no-docker`. 
```

Comment:
I think we can combine the two lines so as to minimize the amount we repeat ourselves.

```suggestion
    * To inspect the script locally before running, run `curl https://install.terraform.io/ptfe/stable > install.sh` and, once you are satisfied with the script's content, execute it with `sudo bash install.sh`. If preferred, the automated installation of Docker can by bypassed by executing the script as `sudo bash install.sh no-docker`.
```

Reply:
I disagree. RHEL requires specific steps that differ from other platforms. 'If preferred' infers not required, which is incorrect, and doesn't specifically identify the step as required for RHEL. The confusion/issues customers experience stems from us not being explicitly clear in our docs for RHEL, as well as with other areas (I have other forthcoming docs edits).